pkg/api/volume/handler.go

```diff
@@ -95,32 +96,58 @@ func (h *ActionHandler) Do(ctx *harvesterServer.Ctx) (interface{}, error) {
 	}
 }
 
-func (h *ActionHandler) validateExportVolume(storageClassName, pvcNamespace, pvcName string) string {
-	if storageClassName == "" {
-		return "Parameter `storageClassName` is required"
+func (h *ActionHandler) checkPVCInUsed(pvcNamespace, pvcName string) error {
```

Comment:
Change function name to

```suggestion
func (h *ActionHandler) checkPVCIsUsed(pvcNamespace, pvcName string) error {
```

or

```suggestion
func (h *ActionHandler) checkPVCInUse(pvcNamespace, pvcName string) error {
```

Actually a boolean value should be returned if the function is called like this.

Alternatively

```suggestion
func (h *ActionHandler) assertPVCNotInUse(pvcNamespace, pvcName string) error {
```

Reply:
Thanks for the suggestion. I use the `assertPVCNotInUse` to keep the return error.